src/spaceone/monitoring/connector/kubernetes_connector/pod_log.py

```diff
@@ -14,35 +15,44 @@ class PodLog(KubeConnector):
     def __init__(self, client=None, **kwargs):
         super().__init__(**kwargs)
 
-        self.prevTime = kwargs.get('start', '')
-        query = kwargs.get('query', {})
-        self.namespace = query.get('namespace', '')
-        self.podName = query.get('podName', '')
-
-    def list_logs(self):
+    def list_logs(self, params):
         try:
+            previous_time = params.get('start', '')
+            query = params.get('query', {})
+            namespace = query.get('namespace', '')
+            pod_name = query.get('name', '')
             result = []
             pod_logs = ""
```

Comment:
You can remove lines 24 and 25. There are no codes to reference these variables.